install.py

```diff
@@ -10,7 +10,18 @@
 import platform
 import warnings
 import datetime
+import importlib
 
+def install_and_import(package):
```

Comment:
* Rename it to `install_python_package` instead, that'd fit with the convention and is more intuitive. 
* Move this function down while maintaining 2 line spacing with the other functions